pages/awell-extensions/marketplace/[...extensionKey].tsx

```diff
@@ -31,28 +31,6 @@ export default function ExtensionPage({ extension }: PageProps) {
         canonicalUrl={`/${Space.AWELL_EXTENSIONS}/marketplace/${extensionKey}`}
       />
       <ExtensionDetail extension={extension} />
```

Comment:
@sharlotta93 The footer is missing here. See http://localhost:3000/awell-extensions/marketplace/cloudinary